machine_learning/linear_discriminant_analysis.py

```diff
@@ -128,23 +128,28 @@ def predict(x_items: list, means: list, variance: float, probabilities: list) ->
     :return: a list containing predicted Y values
     """
 
-    results = []    # An empty list to store generated discriminant values of all items in dataset for each class
+    results = (
+        []
+    )  # An empty list to store generated discriminant values of all items in dataset for each class
```

Comment:
Put this comment on a separate line above so that __black__ does not make your code ugly when it reformats.